api_spec.yaml

```diff
@@ -227,35 +226,33 @@ paths:
         '500':
           $ref: '#/components/responses/500'
 
-  /appid-<appId>/recipe/oauth2/token/revoke:
-    post:
```

Comment:
Post all API merging:
- need to check that there is no status with EXPIRY in it
- need to check that clientSecretRequiredForRefresh is being returned from all APIs that return token info